admin/app.py

```diff
@@ -112,5 +125,28 @@ def restore_user(_id: str):
         return redirect(url_for('get_active_users_list'))
 
 
+@app.route('/admin_login', methods=['GET', 'POST'])
+def admin_login():
+    form = LoginForm()
+    if form.validate_on_submit():
+        admin = User.objects(login=form.admin.data).first()
```

Comment:
@AndriiSkavreniuk Use `User.objects.get(login=form.admin.data)` instead of `User.objects(login=form.admin.data).first()`